docs/docs/parachains/picasso/governance.md

```diff
@@ -0,0 +1,177 @@
+# A look into Picasso Governance
+
+As we plan to completely decentralize the governance on the Picasso network, 
+we have designed the decisions affecting the parachain to go through a fair 
+democratic process supported by the Substrate governance affiliated pallets. 
+The Democracy pallets we are utilizing are provided within the Substrate libraries 
+and a core piece of the logic that constitutes the runtime of the Kusama and Polkadot networks. 
+Referendums will be carried out to ensure agreement from the wider community 
+and to establish transparency when significant changes are made to Picasso.
+
+## Governance Divisions
+
+### Picasso Council
+
+Upon launch, governance on Picasso parachain will be done by a Sudo account held by the Composable. 
+Following the onboarding of core pallets on Picasso, 
+the Sudo council will be burned and the First Council will then be in charge of governance. 
+The First Council is an on-chain entity made up of 11 senior team members from and our supporters. 
+The Council members also control Picasso’s multi-sig wallet.
+
+When the Picasso parachain is live, each council member will be represented as an on-chain account on Polkadot.js. 
+Members of the First Council will consist of:
+- 0xBrainJar, Composable Founder/CEO
+- Jeff Smith, Composable General Counsel (GC)
+- Eoin Whelan, Composable CFO
+- Seun Lanlege, Composable Rust Substrate Lead
```

Comment:
bridging protocol lead

Reply:
According to the content update doc the polkassembly guide isn’t ready(?) It’s marked as pending review.